src/Navigation/Navigation.js

```diff
@@ -100,5 +101,6 @@ export const Navigation = () => (
       component={VideoPage}
       options={({ route }) => ({ title: route.params.video.title })}
     />
+    <Stack.Screen name="FitnessPage" component={FitnessPage} options={({ route }) => ({ title: 'title fitness' })} />
```

Comment:
Should have a nicer title, even if overridden

Reply:
c'est vraiment important ?